tests/integration/wdio.docker.js

```diff
@@ -24,7 +24,7 @@ exports.config = Object.assign({}, common.config, {
     },
     options: {
       e: ["MOZ_HEADLESS=1",
-          "MAILINATOR_PASSWORD=${MAILINATOR_PASSWORD}",
+          "MAILINATOR_PASSWORD=${MONITOR_FXA_PASSWORD}",
```

Comment:
And I _think_ this is OK here? Seems almost like unrelated passwords (mailinator disposable passwords vs a potentially sensitive MONITOR_FXA_PASSWORD)? But considering the following two API keys are probably much more secret, I'm guessing there isn't a way for anybody to ssh or pull a docker image and get those secrets and be naughty(tm).

Reply:
Yeah, this password is for a test account on https://stable.dev.lcip.org/. So even if someone got this, I think the worse they could do is to DoS that account which would break our tests. :/

Reply:
I highly doubt anyone would crack that password.